src/options.cpp

```diff
@@ -3110,6 +3118,7 @@ bool options_manager::save()
     const auto savefile = PATH_INFO::options();
 
     update_options_cache();
+    memorized_tile_count = ::get_option<int>( "MEMORIZED_TILE_COUNT" );
```

Comment:
Is this really the right place to be setting this variable?  We have several other options which are cached in global variables, and none of them are being set here.  It feels like they should be handled consistently.

Reply:
I backported this directly from BN's codebase, so as far as proper usage goes, I'm kinda blind. I'm open to suggestions, though.